tests/commands/test_unset.py

```diff
@@ -8,32 +8,31 @@
          'lap top_ap ple_mac book pro']
 
 directories = ['.',
-               'simple',
                's p a c e s',
-               's p a/c e s',
                'r/e/c/u/r/s/i/v/e',
                'overlap/one',
                'overlap/two',
-               'very/very/very/deep']
+               'very/very/very/deep',
+               ]
 
 assets = [f"{d}/{f}.{i}" for f in files for i, d in enumerate(directories)]
 
+content_dict = {"one": "value",
```

Comment:
I thought I left a comment on this before, but perhaps it didn't save.

I would prefer that both the keys and values here are predictable. Currently the keys are, but the values aren't. There's currently no clear, predictable relation between them.

Reply:
You did, and yes, I did it just for the keys. It will be updated with the next push.